temoa/model_checking/unit_checking/table_checker.py

```
@@ -0,0 +1,129 @@
+                    converted_units.append(unit_obj)
+
+        # assemble a reference of item: units-relationship if we have a valid entry
+        if len(converted_units) == format_type.groups:  # we have the right number
+            if format_type == SINGLE_ELEMENT:
+                ref = {expr: converted_units[0]}
+                res.update(ref)
+            elif format_type == RATIO_ELEMENT:
+                ref = {expr: converted_units[0] / converted_units[1]}
+                res.update(ref)
+            else:
+                raise ValueError('Unknown units format: %s', format_type)
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)